homeassistant/components/matter/light.py

```diff
@@ -351,11 +357,21 @@ def _update_from_device(self) -> None:
             self._attr_brightness = self._get_brightness()
 
 
+class ColorControlCapabilities(IntFlag):
+    """Color control capabilities bitmap."""
+
+    HS = 0x1
+    EHUE = 0x2
+    CL = 0x4
+    XY = 0x8
+    TEMP = 0x10
```

Comment:
use descriptive variable names, better to write "COLOR_TEMPERATURE" than "COLOR_TEMP" or just "TEMP".
Doesn't matter in this case because this enum should not exist in the first place.